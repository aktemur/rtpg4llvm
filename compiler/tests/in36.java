/*compunit*/
int* foo(int x, double y) { }
