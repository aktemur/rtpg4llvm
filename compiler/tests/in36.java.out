new FuncDeclaration(new PtrType(new IntType()),
      "foo",
      mkFormals(2, new FuncParam(new IntType(), "x"),
         new FuncParam(new DoubleType(), "y")),
      mkScopedStmt(0))
